packages/components/src/button/stories/best-practices.mdx

```diff
@@ -0,0 +1,65 @@
+
+
+## Usage
+Buttons indicate available actions and allow user interaction within the interface. As key elements in the WordPress UI, they appear in toolbars, modals, and forms. Default buttons support most actions, while primary buttons emphasize the main action in a view. Secondary buttons pair as secondary actions next to a primary action. 
+
+Each layout contains one prominently placed, high-emphasis button. If you need multiple buttons, use one primary button for the main action, secondary for the rest of the actions and tertiary sparingly when an action needs to not stand out at all.
+
+## Variants
+
+### Primary
+Primary buttons stand out with bold color fills, making them distinct from the background. Since they naturally draw attention, each layout should contain only one primary button to guide users toward the most important action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-640&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="primary" height="400" />
+
+### Secondary
+Secondary buttons complement primary buttons. Use them for standard actions that may appear alongside a primary action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-808&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="secondary" height="400" />
+
+### Tertiary
+Tertiary buttons have minimal emphasis. Use them sparingly to subtly highlight an action.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-874&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="tertiary" height="400" />
+
+### Default
+Use default buttons for actions of equal priority.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-940&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="default" height="400" />
+
+### Link
+Link buttons have low emphasis and blend into the page, making them suitable for supplementary actions, especially those involving navigation away from the current view.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-1006&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="link" height="400" />
+
+### Destructive
+Use this variant for irreversible actions. Apply sparingly and only for actions with significant impact.
+
+<FigmaEmbed url="https://www.figma.com/design/V7NfxdP0ybEMXf01WyWGKB/Storybook-documentation-Figma-assets?node-id=5-1069&node-type=frame&t=HfUDxJShbqa3dM4N-11" title="destructive" height="400" />
+
+### Sizes
+The default size is 40px. Use smaller sizes for compact views or when placing buttons in small components, such as dropdowns or dialogs.
```

Comment:
This seems vague, compared to the more specific guidelines already given in the [props docs for `size`](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/button/README.md#size-default--compact--small). I think people will need the more specific guidance to design consistently.

Reply:
I'll let @jameskoster chime in here. My interpretation was that these size assignments were for the older version of this component, but I could be totally wrong.

Reply:
Agree it's vague, but I'm not sure we can do much better until there are designs for more admin pages/patterns. At the moment smaller button sizes really are just for use in UIs that require greater density. I suppose we could add a note saying that if you're unsure then use the default size, but I don't know if it's worth it?

Reply:
I'm going to resolve this one for now. I'm sure we'll get to a better set of recommendations as we iterate on button philosophy and general documentation best practices. Thank you for making these changes @mirka 🙏